addons/l10n_es_edi_facturae/__manifest__.py

```diff
@@ -8,8 +8,8 @@
     'description': """
     This module create the Facturae file required to send the invoices information to the General State Administrations.
     It allows the export and signature of the signing of Facturae files.
-    The current version of Facturae supported is the 3.2.1
-    
+    The current version of Facturae supported is the 3.2.2
+
```

Comment:
Was there a reason for that one?
As most of the software/systems used for facturae do not handle 3.2.2 modifications (at least last time I checked) That was the reason we chose 3.2.1 and not 3.2.2 in the first place

Reply:
@JulienAlardot No specific reason apart from noticing there was a more up-to-date version. I didn't know the choice for 3.2.1 was because of compatibility issues. I guess we could change it back, since pretty much nothing changed structurally. What do you think @chklop?

Reply:
Time to evolve honestly, also are the changes that big ?

Reply:
Nope, pretty much nothing. Leaving it like this then ;)

Reply:
No, just a new unit of measure and a few optional things
I was just wondering if it was intended
Note that the official facturae software isn't able to load these